src/main/java/org/mastodon/mamut/tomancak/lineage_registration/LineageRegistrationUtils.java

```diff
@@ -35,9 +35,8 @@
 	 * The sorting is based on the result of the
 	 * {@link LineageRegistrationAlgorithm}.
 	 */
-	public static void sortSecondTrackSchemeToMatch( Model modelA, Model modelB )
+	public static void sortSecondTrackSchemeToMatch( Model modelA, Model modelB, RegisteredGraphs result )
```

Comment:
I see your point regarding the signature.
However, keeping the unused parameter may cause that developers wonder why this method contains a param that is actually not used in the future. 

If you really want to keep it, like this, I would suggest the following javadoc:

`/**
	 * Sorts the descendants in the second {@link ModelGraph} to match the order
	 * of the descendants in the first {@link ModelGraph}.
	 * The sorting is based on and thus needs the result of the registration of both {@link ModelGraph}s using the
	 * {@link LineageRegistrationAlgorithm}.
	 *
	 * @param modelA the first {@link ModelGraph}, i.e. the one that serves as reference. 
	 * @param modelB the second {@link ModelGraph}, i.e. the one that is sorted.
	 * @param registration the mapping between the two {@link ModelGraph}s (the result of the {@link LineageRegistrationAlgorithm})
	 */`

However, I think, it would be even better, if this method had the signature `sortSecondTrackSchemeToMatchFirst(Model firstModel, Model secondModel)` and would run the registration internally, without the need to have the registration result as an input param. In this case, both params would be really needed an the method signature would also be clearer.
In order to not repeat the registration each time this method would be called, on may consider to move this method to the class `RegisteredGraphs`.

Reply:
Did you consider the above suggestions in #28?
I cannot find any changes related to them in #28.